cdk/csharp/src/Cdk.sln

```diff
@@ -0,0 +1,48 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.26124.0
+MinimumVisualStudioVersion = 15.0.26124.0
```

Comment:
Is this a common value? Is it worth documenting in the README?

I have no idea about .NET development best practices 😄

Reply:
This is a standard solution file defining the projects. 
All common values, see here : https://learn.microsoft.com/en-us/visualstudio/extensibility/internals/solution-dot-sln-file?view=vs-2022#file-header
I don't think we need to document this.